src/messages/layouts/MessageLayout.cpp

```diff
@@ -376,11 +376,22 @@ void MessageLayout::updateBuffer(QPixmap *buffer,
              this->message_->flags.has(MessageFlag::LowTrustUsers))
     {
         backgroundColor = QColor("#404040");
+        if (this->message_->flags.has(MessageFlag::AutoModOffendingMessage) &&
+            ctx.preferences.enableAutomodHighlight)
+        {
+            backgroundColor =
+                *ctx.colorProvider.color(ColorType::AutomodHighlight);
+        }
```

Comment:
Not sure if I should blend colors here

Reply:
Yes, they should be blended, but with the original background color - not the automod color. Previously, they weren't blended, as `#404040` is fully opaque. Now, users can set a color with transparency which would blend with the default color of a `QPixmap` - the backing buffer for a message (that one is actually undefined but usually some light gray with MSVC).

You could put the assignment to the default color in the `else` branch.

Reply:
Got it, I think it's fixed now.